docs/containers/quickstart-container-registries.md

```diff
@@ -94,7 +94,8 @@ For each tagged image in a repository, here are the actions that can be performe
 - **Pull Image**: copies the latest version of the image locally
```

Comment:
Should the image on line 92 be updated?

Reply:
Yes this should be updated to include the new item about ACA